openverse_catalog/dags/maintenance/check_silenced_dags.py

```diff
@@ -114,9 +121,19 @@ def check_configuration(github_pat: str):
 
 with dag:
     PythonOperator(
-        task_id="check_silenced_dags_configuration",
+        task_id="check_silenced_notifications_configuration",
+        python_callable=check_configuration,
+        op_kwargs={
+            "github_pat": "{{ var.value.get('GITHUB_API_KEY', 'not_set') }}",
+            "airflow_variable": "SILENCED_SLACK_NOTIFICATIONS",
+        },
+    )
+
+    PythonOperator(
+        task_id="check_skipped_errors_configuration",
         python_callable=check_configuration,
         op_kwargs={
             "github_pat": "{{ var.value.get('GITHUB_API_KEY', 'not_set') }}",
+            "airflow_variable": "SKIPPED_INGESTION_ERRORS",
         },
     )
```

Comment:
Do you think we could set this up as a for loop to generate these tasks since they're so similar? 😄

Reply:
Sure, we can have the `task_id` be generated from the name of the Airflow variable being checked 👍